### DISCUSSION TITLE
Running LAMMPS code

### DISCUSSION THREAD

**Discussion by jainaarushi** [deepmodeling/deepmd-kit#941] [Q&A]
I want to run LAMMPS with the latest version of Deepmd. I have installed Deepmd through the python interface. What is the procedure for it?

**Comment by njzjz** [comment]
Please read our documents for C++ interface installation.
Stable version: https://docs.deepmodeling.org/projects/deepmd/en/master/
Development: https://docs.deepmodeling.org/projects/deepmd/en/latest/
